wrapper/src/template.sh

```diff
@@ -1,6 +1,12 @@
 #!/usr/bin/env bash
 
 {
+    if [ -z "$BATECT_QUIET_DOWNLOAD" ]; then
```

Comment:
How about something like:

```bash
BATECT_QUIET_DOWNLOAD=${BATECT_QUIET_DOWNLOAD:-false}
```

This should also go down with the other variables (eg. around line 23)

Reply:
Whoa that's a neat trick I didn't know about!